thirdeye-ui/src/app/components/rca/top-contributors-section/top-contributors-section.component.tsx

```diff
@@ -83,7 +84,15 @@ export const TopContributorsSection: FunctionComponent<TopContributorsSectionPro
         useEffect(() => {
             notifyIfErrors(
                 anomalyDimensionAnalysisReqStatus,
-                errorMessages,
+                generateCustomErrorMessage(errorMessages, [
+                    {
+                        criteriaFunction: (error) =>
+                            error
+                                .toLowerCase()
+                                .includes("pinotclientexception"),
```

Comment:
this will get broken when we change error management on the backend side. 
the frontend should only match on 
the `code` field which is part of the API.

Eg: 
```
code: "ERR_ALERT_PIPELINE_EXECUTION"
```

Let me check if it's easy to introduce a new code error for Pinot exceptions.

Reply:
Makes sense! Can you send the exact code to match 2 kinds of errors
- Error with Pinot broker
- Timeout error (assuming this exception is thrown by the backend and not rely on browser timeout)
?

Reply:
There are no such code yet. You can mark the ticket as blocked and mention me. I won't be able to work on this today.